README.md

```diff
@@ -19,4 +19,8 @@ The endpoint for the API is: <https://api.napari.org>
 
 ## Contributing
 
-If you are interested in contributing to this repo, see the [CONTRIBUTING.md](./CONTRIBUTING.md) page.
\ No newline at end of file
+If you are interested in contributing to this repo, see the [CONTRIBUTING.md](./CONTRIBUTING.md) page.
+
+## Code of conduct
+
+`npe2api` is maintained by the [napari](https://napari.org/stable/) community. The `napari` community has a [Code of Conduct](https://napari.org/stable/community/code_of_conduct.html) that should be honored by everyone who participates in the `napari` community.
```

Comment:
```suggestion
`npe2api` is maintained by the [napari](https://napari.org/) community. The `napari` community has a [Code of Conduct](https://napari.org/dev/community/code_of_conduct.html) that should be honored by everyone who participates in the `napari` community.
```

Updated since we are moving away from our unversioned pages workflow, and the code of conduct that applies should be the latest version.